packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -229,6 +228,98 @@ describe("IntervalCollection fuzz testing", () => {
 	});
 });
 
+// currently fails
+describe.skip("IntervalCollection offline fuzz testing", () => {
```

Comment:
I was going to comment that it might be nicer to put these in the same suite, but I realized `.skip`ing seeds isn't possible right now. That's pretty easy to implement: #16270